src/cl-gists.lisp

```diff
@@ -1,6 +1,7 @@
 (in-package :cl-user)
 (defpackage cl-gists
   (:use :cl)
+  (:nicknames :gists)
```

Comment:
is this strictly necessary?

Reply:
@jhunt, not strictly necessary for operation and there as a syntactic convenience. Most programmers will know what language they are working in, and the `cl-`  prefix is redundant.